services/order.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from django.db import transaction
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
The `create_order` function does not handle the case where a user with the specified username does not exist. Consider using a try-except block to catch `User.DoesNotExist` exceptions and handle them appropriately.